teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>Since the table columns are fixed, we could use indices to slice the DataFrame instead of column names (would be shorter)?</p><p> </p><p>Again, I would move the comment to the Markdown cell above (same for the following cells, will not comment again to not bore you :))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Markdown, done.</p><p> </p><p>@dominiquesydow: Concerning the indices I'm undecided. It would be shorter, but it might be harder to understand what happened?</p>

Reply:
@AndreaVolkamer: Would you be ok with selecting here the columns (by name) that we want - instead of omitting those we do not? Would be less columns to pick and it would introduce already the data columns that we look at afterwards.

Reply:
Suggesting instead of
```python
col_to_drop = ['ALA','ARG','ASN','ASP','CYS','GLN','GLU','GLY','HIS','ILE','LEU','LYS','MET','PHE','PRO','SER','THR','TRP','TYR','VAL','lid/hull','ellVol', 'Cs','Ns','Os','Ss','Xs']
metadata.drop(col_to_drop, axis=1, inplace=True)
```
to use
```python
metadata = metadata[
    [
        'lig_cov', 
        'poc_cov', 
        'lig_name', 
        'volume', 
        'enclosure', 
        'surface', 
        'depth', 
        'surf/vol', 
        'accept', 
        'donor', 
        'hydrophobic_interactions', 
        'hydrophobicity', 
        'metal', 
        'simpleScore', 
        'drugScore'
    ]
]
```
(I did not select here the ell* and *AA columns, since we do not use them - we could deselect even more if you want.)

Reply:
Perfect! Good idea to run it around!